src/App.js

```diff
@@ -1,22 +1,34 @@
 import React from 'react';
 import './App.css';
 
+import TodoList from './components/todo-list';
 import todos from './api/todos';
 import users from './api/users';
 
+const preparedTodos = todos.map((todo) => {
+  const user = users.find(u => u.id === todo.userId);
```

Comment:
```suggestion
  const user = users.find(person => person.id === todo.userId);
```
нейминг переменных должен быть осмысленным